app/controllers/api/v2/access_requests_controller.rb

```diff
@@ -16,11 +16,24 @@ def index
         render jsonapi: @access_requests
       end
 
+      def create
+        authorize AccessRequest # todo, generalise admin auth
+        access_request = AccessRequest.new(access_request_params)
+        access_request.requester = @current_user
+        access_request.request_date_utc = Time.now.utc
+        access_request.status = :requested
```

Comment:
if you do 
` access_request.update(status: 'requested')
it updates the attribute then saves.

Reply:
So you could do 
``
access_request.update(status: 'requested', request_date_utc: Time.now.utc, requester: @current_user)
``

Reply:
done, nice one